File: features/user/groupchat/src/main/java/com/tokopedia/groupchat/chatroom/domain/pojo/channelinfo/ChannelInfoPojo.java

package com.tokopedia.groupchat.chatroom.domain.pojo.channelinfo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ChannelInfoPojo {

    @SerializedName("channel")
    @Expose
    private Channel channel;

    public Channel getChannel() {
        return channel;
    }

    public void setChannel(Channel channel) {
        this.channel = channel;
    }

}

File: features/user/groupchat/src/main/java/com/tokopedia/groupchat/chatroom/kotlin/domain/pojo/channelinfo/ChannelInfoPojo.kt
package com.tokopedia.groupchat.chatroom.kotlin.domain.pojo.channelinfo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class ChannelInfoPojo (

    @SerializedName("channel")
    @Expose
    var channel: Channel? = null
){
}
